staff/lab/ocf-tv

```diff
@@ -135,6 +131,17 @@ def check_volume(val):
     return val
 
 
+def update_volume(amount, host):
+    cmd = ''
+
+    if amount:
```

Comment:
You may still wish to use `if amount is not None` here, lest funny things happen when a user tries to set the volume to 0 in interactive mode.

Reply:
For that matter, it may be time to consider separating the functionality of setting volume from listing volume. `update_volume` is a misleading name if you're not updating the volume.